### DISCUSSION TITLE
Is it possible to get same visitorId for different browsers on the same machine (with relative accuracy)

### DISCUSSION THREAD

**Discussion by qiulang** [fingerprintjs/fingerprintjs#687] [Q&A] [answered]
Hi is it possible to get a same visitorId for different browsers on the same machine? I just need a relative accuracy, say 90% sure they are from the same machine just one from chome and another from firefox/IE.

I notice this issue was opened/closed in 2015 https://github.com/fingerprintjs/fingerprintjs/issues/10 and it said no

But I was wondering if that is possbile in 2021 ?

Thanks

Qiulang

**Comment by makma** [comment, marked as answer]
Hi, @qiulang!
No, this is not the use case for this library. The result fingerprint/hash is generated from the unique signals from the browser, not the machine (hence browser fingerprinting).

**Reply by qiulang** [reply]
Thank you for the clarification.

**Reply by maucaro** [reply]
I am getting the same visitorid across Chrome and Edge on the same machine. In my use case, I'd actually like these to be different. Is it perhaps because they are both using the Chromium engine? Is there a way to make them different?

**Reply by Valve** [reply]
@maucaro if you are getting the same visitorID values on different browsers, please submit an issue with all the details to help us take a deeper look into it. 
Thanks

**Reply by maucaro** [reply]
Will do. Thanks!

**Reply by hataptys** [reply]
Was an issue submitted for this?
I am seeing the same issue. Using version 3.3.0 in our software but also tried version 3.4.0 from https://www.npmjs.com/package/@fingerprintjs/fingerprintjs and receiving the same Visitor ID between Chrome and Edge browsers.
Thank you.

**Reply by makma** [reply]
@hataptys please take a look at https://github.com/fingerprintjs/fingerprintjs/issues/756
